app/views/comments/_comment_date.erb

```diff
@@ -1,7 +1,15 @@
 <div class="comment-date">
   <a href="<%= decorated_comment.path %>">
+  <span class="hidden m:inline-block">
     <time datetime="<%= decorated_comment.published_timestamp %>">
       <%= decorated_comment.readable_publish_date %>
     </time>
+    </span>
   </a>
-</div>
+  <% if !decorated_comment.edited_at? %>
+    •
+    <span title="<%= decorated_comment.edited_at.strftime('%A, %d %B %Y, %T') %>">
```

Comment:
This line needs changing to 

```erb
<time datetime="<%= decorated_comment.edited_at %>">
```

and the closing span needs to be updated to match.

Reply:
Oh okay, You mean line 11 right?

Reply:
Yep, line 11 and the closing span on line 13 to  `</time>`